lib/debt_ceiling/debt.rb

```diff
@@ -4,36 +4,54 @@ class Debt
     extend Forwardable
     DoNotWhitelistAndBlacklistSimulateneously = Class.new(StandardError)
 
-    attr_reader :file_attributes
     def_delegators :file_attributes, :path, :analysed_module, :module_name, :linecount, :source_code
-    def_delegator :analysed_module, :rating
-    attr_accessor :debt_amount
-    def_delegator :debt_amount, :to_i
+    def_delegator  :analysed_module, :rating
```

Comment:
You're right!

On Fri, Mar 20, 2015 at 3:16 PM, Brian Glusman <notifications@github.com>
wrote:

> In lib/debt_ceiling/debt.rb
> <https://github.com/bglusman/debt_ceiling/pull/17#discussion_r26877146>:
>
> >      def_delegators :file_attributes, :path, :analysed_module, :module_name, :linecount, :source_code
> > -    def_delegator :analysed_module, :rating
> > -    attr_accessor :debt_amount
> > -    def_delegator :debt_amount, :to_i
> > +    def_delegator  :analysed_module, :rating
>
> def_delegators is for assigning a bunch of methods to one object... this
> is assigning rating to analysed_module, which is in turn delegated to
> file_attributes... so either that's not an option, or I'm misunderstanding
> your question :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bglusman/debt_ceiling/pull/17/files#r26877146>.
>